packages/timeline-state-resolver/src/integrations/tricaster/triCasterStateDiffer.ts

```diff
@@ -305,10 +358,13 @@ export class TriCasterStateDiffer {
 		transitionDuration: this.durationCommandGenerator,
 		...this.layerCommandGenerator,
 		input: CommandName.SELECT_NAMED_INPUT,
-		onAir: ({ state, target }) => {
+		onAir: ({ state, target, value }) => {
 			if (state.transitionEffect === 'cut') {
-				return [{ name: CommandName.TAKE, target }]
+				return [{ name: CommandName.VALUE, target, value: value ? 1 : 0 }]
```

Comment:
I take that wherever this is called, it expects `value` to be of type `number` and not `boolean`?

If the value of `value` is ALWAYS suppose to be `1` or `0`, then it could be written as `Number(!!value)`.
I am not saying that is a better way to write it, just suggesting an alternative to a ternary, that from the looks of it just gets the number representation of the boolean outcome.